src/components/LeaguesTab/LeaguesTab.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import LeaguesTabStyle from './LeaguesTabStyle';
+import LeaguesTabItem from 'Components/LeaguesTabItem';
```

Comment:
`LeaguesTabItem` -> `LeagueTab`

Reply:
it was set in webpack.config.js with the help of alias, i've just typed it with uppercase to quickly see the difference between common paths and those ones with aliases

Reply:
It is not clear, your code should be understandable. I think it doesn't matter what the differences between paths, it doesn't help in codding.